<ui version="4.0" >
 <class>UPnPWidget</class>
 <widget class="QWidget" name="UPnPWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UPnP</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Detected devices:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_devices" >
     <column>
      <property name="text" >
       <string>Device</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Port Forwarded</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>WAN Connection</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="m_forward" >
       <property name="text" >
        <string>Forward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_undo_forward" >
       <property name="text" >
        <string>Undo Forward</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>271</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_rescan" >
       <property name="text" >
        <string>Rescan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
